codex-rs/core/src/windows.rs

```diff
@@ -0,0 +1,250 @@
+#[cfg(target_os = "windows")]
```

Comment:
Just calling this file `windows.rs` is quite broad. Please come up with a more specific purpose.

Then, rather than having to put `#[cfg(target_os = "windows")]` everywhere, in the `lib.rs` for this crate, I would favor using ``#[cfg(target_os = "windows")]`` once before `mod windows;` in `lib.rs`.